Cargo.toml

```diff
@@ -30,17 +30,25 @@ path = "src/lib.rs"
 [dependencies]
 inherent = "1.0"
 sea-query-derive = { version = "0.4.2", path = "sea-query-derive", default-features = false, optional = true }
-serde_json = { version = "1", default-features = false, optional = true, features = ["std"] }
-chrono = { version = "0.4.27", default-features = false, optional = true, features = ["clock"] }
+serde_json = { version = "1", default-features = false, optional = true, features = [
```

Comment:
Please, revert changes in the `Cargo.toml`

Reply:
thanks for taking a look! I understand that the formatting changes need to be reverted. But i added a new dependency `strum_macros` (which is already a depdency of sea-orm), i should keep it there, right?

Reply:
@ikrivosheev Made the request changes. Could you please review it again?